plugins/tiddlywiki/multiwikiserver/modules/store/sql-tiddler-database.js

```diff
@@ -10,673 +10,678 @@ Validation is for the most part left to the caller
 
 \*/
 
-(function() {
+(function () {
```

Comment:
There is a prettyprinting problem here. We do use TAB 4 as indentation in TW. 
And there is no space between `function()`